homeassistant/config_entries.py

```diff
@@ -881,6 +891,21 @@ def async_create_background_task(
         task.add_done_callback(self._background_tasks.remove)
         return task
 
+    async def _async_load_translations(self, hass: HomeAssistant) -> None:
+        """Load translations."""
+        config_language = hass.config.language
+        try:
+            self.translations = await translation.async_get_translations(
+                hass, config_language, "device", {self.domain}
+            )
+        except Exception as err:  # pylint: disable=broad-exception-caught
+            _LOGGER.debug(
+                "Could not load translations for %s",
+                self.domain,
+                exc_info=err,
+            )
+            self.translations = {}
```

Comment:
Maybe it should be clearer this is only loading device translations

Reply:
This is no longer relevant since we no longer store a reference to translations in the config entry